sw/device/lib/crypto/drivers/kmac.h

```diff
@@ -39,11 +59,40 @@ typedef enum kmac_operation {
  */
 typedef enum kmac_error {
   kKmacOk = 0,
-  kKmacInternalError = 1,
-  kKmacArgsError = 2,
-  kKmacNotImplemented = 3,
+  kKmacInternalError,
+  kKmacNotIdle,
+  kKmacCfgWriteDisabled,
+  kKmacIntrErrPending,
+  kKmacCfgDisabledError,
+  kKmacFatalFaultError,
+  kKmacRecovFaultError,
+  kKmacArgsError,
+  kKmacDigestLenTooLongError,
+  kKmacUnsupportedKeySizeError,
+  kKmacNotImplemented,
+  kKmacCustomPaddingError,
+  kKmacUnsupportedPaddingLength,
 } kmac_error_t;
 
+/*
+ * Set the "global" config of HWIP
+ *
+ * For the moment, we have a number of configuation options needs to be
+ * configured (probably) at session level. This functions serves as a temporary
```

Comment:
```suggestion
 * configured at session level. This functions serves as a temporary
```
Not even probably :slightly_smiling_face: